static/src/javascripts/projects/common/modules/ui/subscription-medium-banner.js

```diff
@@ -0,0 +1,242 @@
+// @flow
```

Comment:
Would be wonderful to see more flow-strict 🙏 😸 

Using Flow Strict will enable stronger safety guarantees in Flow
https://flow.org/en/docs/strict/